digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/MunicipalityDao.scala

```diff
@@ -41,6 +41,7 @@ class MunicipalityDao {
       select name_fi from municipality where id = $id""".as[String].first
   }
 
+  //Todo Review this query
```

Comment:
I think this was decided that is useful, since it checks if the municipality ID exists on the database. The TODO comment can be removed.